Bottom: a4e302c3ad7b2bb0dc1cee5b1e508bc67165b949
Top:    477303bbeccdf3cd3799b5729428253be585b44a
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-07-16 21:19:59 -0700

Refresh of librdmacm-3-8-autogen-sh-use-a

---

diff --git a/autogen.sh b/autogen.sh
index 6c9233e..24c0a76 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -1,4 +1,5 @@
 #! /bin/sh
 
 set -x
+test -d ./config || mkdir ./config
 autoreconf -ifv -I config
